.werft/jobs/build/deploy-to-preview-environment.ts

```diff
@@ -78,7 +78,7 @@ export async function deployToPreviewEnvironment(werft: Werft, jobConfig: JobCon
     const domain = withVM ? `${destname}.preview.gitpod-dev.com` : `${destname}.staging.gitpod-dev.com`;
     const monitoringDomain = `${destname}.preview.gitpod-dev.com`;
     const url = `https://${domain}`;
-    const imagePullAuth = exec(`printf "%s" "_json_key:$(kubectl get secret ${IMAGE_PULL_SECRET_NAME} --namespace=keys -o yaml \
+    const imagePullAuth = exec(`printf "%s" "_json_key:$(kubectl --kubeconfig ${CORE_DEV_KUBECONFIG_PATH} get secret ${IMAGE_PULL_SECRET_NAME} --namespace=keys -o yaml \
```

Comment:
What happens if you remove `${CORE_DEV_KUBECONFIG_PATH}`? We have different kubectx (configs) for different types of preview environments, it seems risky to add because the other type (`with-vm`) may stop working.

Reply:
> What happens if you remove `${CORE_DEV_KUBECONFIG_PATH}`?

It'd render an empty secret.

Good catch! I'll check if other kubeconfigs. I use the `${CORE_DEV_KUBECONFIG_PATH}` because other code references to use this kubeconfig.
- https://github.com/gitpod-io/gitpod/blob/8dd76ed762784004c1b734beb286410037306794/.werft/jobs/build/deploy-to-preview-environment.ts#L104
- https://github.com/gitpod-io/gitpod/blob/8dd76ed762784004c1b734beb286410037306794/.werft/jobs/build/deploy-to-preview-environment.ts#L108

Reply:
Also, can you build a preview environment with `with-vm=true`? That is different from `core-dev`, and will be good to test...to see how this version of `kubeconfig` works.